pkg/systemd/reporting.jsx

```diff
@@ -67,38 +68,25 @@ function get_problem_properties(problem) {
                                     "GetAll", ["org.freedesktop.Problems2.Entry"]));
 }
 
-class FAFWorkflowRow extends React.Component {
-    constructor(props) {
-        super(props);
+const FAFWorkflowRow = ({ problem }) => {
+    const [problemState, setProblemState] = React.useState(ProblemState.REPORTABLE);
+    const [process, setProcess] = React.useState(null);
+    const [reportLinks, setReportLinks] = React.useState([]);
+    const [message, setMessage] = React.useState("");
 
-        this.state = {
-            problemState: ProblemState.REPORTABLE,
-            process: null,
-            reportLinks: [],
-            message: "",
-        };
-
-        this._onCancelButtonClick = this._onCancelButtonClick.bind(this);
-        this._onReportButtonClick = this._onReportButtonClick.bind(this);
-        this.updateStatusFromBus = this.updateStatusFromBus.bind(this);
-
-        this.updateStatusFromBus();
-    }
-
-    updateStatusFromBus() {
-        get_problem_properties(this.props.problem.path)
+    const updateStatusFromBus = () => {
+        get_problem_properties(problem.path)
                 .catch(exception => {
-                    this.setState({ problemState: ProblemState.UNREPORTABLE });
-
-                    console.error(cockpit.format("Getting properties for problem $0 failed: $1", this.props.problem.path, exception));
+                    setProblemState(ProblemState.UNREPORTABLE);
+                    console.error(cockpit.format("Getting properties for problem $0 failed: $1", problem.path, exception));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19403-20231002-082424-b39d5ff6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)